CONTRIBUTION.md

```diff
@@ -33,7 +34,223 @@ These commands can be executed from the project root:
+			table.AddRow(*resource.ResourceId, *resource.Name, *resource.State)
+		}
+		err := table.Display(cmd)
+		if err != nil {
+			return fmt.Errorf("render table: %w", err)
+		}
+		return nil
+	}
+}
+```
+
+Please remeber to always add unit tests for `parseInput`, `buildRequest` (in `bar_test.go`), and any other util functions used.
+
+If the new command `bar` is the first command in the CLI using a STACKIT service `foo`, please refer to [Onboarding a new STACKIT service](./CONTRIBUTION.md/#onboarding-a-new-stackit-service).
+
+#### Outputs, prints and debug logs
+
+The CLI has 4 different verbosity levels:
+
+- `error`: For only displaying errors
+- `warning`: For displaying user facing warnings _(and all of the above)_
+- `info` (default): For displaying user facing info, such as operation success messages and spinners _(and all of the above)_
+- `debug`: For displaying structured logs with different levels, including errors _(and all of the above)_
+
+For prints that are specific to a certain log level, you can use the methods defined in the `print` package: `Error`, `Warn`, `Info`, and `Debug`.
+
+For command outputs that should always be displayed, no matter the defined verbosity, you should use the `print` methods `Outputf` and `Outputln`.
+
+### Onboarding a new STACKIT service
+
+If you want to add a command that uses a STACKIT service `foo` that was not yet used by the CLI, you will first need to implement a few extra steps to configure the new service:
+
+1.  Setup the SDK client configuration, using the authentication method configured in the CLI
+
+    1.  This is done in `internal/pkg/services/foo/client/client.go`
+    2.  Below is an example of a typical `client.go` file structure:
+
+        ```go
+        package client
+
+        import (
+           (...)
+           "github.com/stackitcloud/stackit-sdk-go/services/foo"
+        )
+
+        func ConfigureClient(cmd *cobra.Command) (*foo.APIClient, error) {
+           var err error
+           var apiClient foo.APIClient
+           var cfgOptions []sdkConfig.ConfigurationOption
+
+           authCfgOption, err := auth.AuthenticationConfig(cmd, auth.AuthorizeUser)
+           if err != nil {
+              return nil, &errors.AuthError{}
+           }
+           cfgOptions = append(cfgOptions, authCfgOption, sdkConfig.WithRegion("eu01")) // Configuring region is needed if "foo" is a regional API
+
+           customEndpoint := viper.GetString(config.fooCustomEndpointKey)
+
+           if customEndpoint != "" {
+              cfgOptions = append(cfgOptions, sdkConfig.WithEndpoint(customEndpoint))
+           }
+
+           apiClient, err = foo.NewAPIClient(cfgOptions...)
+           if err != nil {
+              return nil, &errors.AuthError{}
+           }
+
+           return apiClient, nil
+        }
+        ```
+
+2.  Add a `FooCustomEndpointKey` key in `internal/pkg/config/config.go` (and add it to `ConfigKeys` and set the to default to `""` using `viper.SetDefault`)
```

Comment:
Steps 2 and 3 should come in the beginning. Otherwise in the client we are using consts that are not yet defined